tests/__init__.py

```diff
@@ -4,9 +4,9 @@
 from contextlib import contextmanager
 from unittest import TestCase
 
-os.environ["REDASH_REDIS_URL"] = os.environ.get("REDASH_REDIS_URL", "redis://localhost:6379/0").replace("/0", "/5")
+os.environ["REDASH_CACHE_URL"] = os.environ.get("REDASH_CACHE_URL", "redis://localhost:6379/0").replace("/0", "/5")
```

Comment:
Nah, we'd better not change this.

It's already documented as `REDASH_REDIS_URL` on the website and has been that way for ages.

So people are probably using this name in things, and changing it will likely create problems. :wink: